domain_substitution.list

```diff
@@ -2831,7 +2831,6 @@ components/safe_search_api/stub_url_checker.cc
 components/safe_search_api/url_checker_unittest.cc
 components/search_engines/default_search_policy_handler_unittest.cc
 components/search_engines/keyword_table_unittest.cc
-components/search_engines/prepopulated_engines.json
```

Comment:
Unless I am missing some context on the changes in Android, this won't help us because of https://github.com/Eloston/ungoogled-chromium/blob/5931f25336c8bbd083a08d564534fbffdc031217/patches/core/ungoogled-chromium/replace-google-search-engine-with-nosearch.patch

Reply:
I think it is probably a slight different problem, that replacing links in that file will prevent users who wants to switch back to Google from using search directly?

Reply:
@Eloston that patch would have to be updated.

I am proposing to replace it **only** in the patch, not via domain substitution; this is also because in Bromite that search engine is allowed.

Reply:
Fair enough. But you need to add this path to `PRUNING_EXCLUDE_PATTERNS` in `devutils/update_lists.py`, otherwise your change will be overridden.

Reply:
@Eloston I have added both exclusions to `DOMAIN_EXCLUDE_PREFIXES` (I think you meant that one)